src/runtime_src/core/pcie/driver/linux/xocl/subdev/xgq_vmr.c

```diff
@@ -1476,9 +1478,37 @@ static int vmr_memory_info_query(struct platform_device *pdev,
 	return vmr_info_query_op(pdev, buf, cnt, XGQ_CMD_LOG_MEM_STATS);
 }
 
+static int xgq_freq_verify(struct platform_device *pdev,unsigned short *target_freqs, int num_freqs)
+{
+	int ret = 0,i;
+	u32 clock_freq_counter, request_in_khz, tolerance, lookup_freq;
+	struct xocl_xgq_vmr *xgq = platform_get_drvdata(pdev);
+	enum data_kind kinds[3] = {FREQ_COUNTER_0,FREQ_COUNTER_1,FREQ_COUNTER_2};
```

Comment:
we should not hardcode kinds here, instead, please check example code:
xocl_clock_get_freq_counter -> clock_counter_get_freq